### PR TITLE
add cadvisor scrape_config

### PR DESCRIPTION
**Pull request by shihchang**
add cAdvisor (Container Advisor) scrape_config to prometheus

### CODE REVIEW

File: THIRD_PARTY_LICENSES.txt [hunk 1]
```diff
@@ -5371,434 +5371,6 @@ LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
 NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-======== Creative Commons License
```
Comment (desagar) [inline THIRD_PARTY_LICENSES.txt:5374]: Why did this file change? I don't see that you changed any dependencies. Did you run attribution-helper for some reason?

Comment (shihchang) [reply THIRD_PARTY_LICENSES.txt:5374]: no go.mod change, but https://build.verrazzano.io/blue/organizations/jenkins/verrazzano-monitoring-operator/detail/scchen%2Fprome_config/21/pipeline
so i need to run attribution-helper which made this THIRD_PARTY_LICENSES.txt change.
Do I miss anything?

File: pkg/resources/helper.go [hunk 2]
```diff
@@ -227,6 +227,25 @@ scrape_configs:
...
    - targets: ["` + pushGWUrl + `"]
+ - job_name: 'cadvisor'
+   scrape_interval: 20s
+   scrape_timeout: 15s
+   kubernetes_sd_configs:
+   - role: node
+   scheme: https
+   tls_config:
+     ca_file: /var/run/secrets/kubernetes.io/serviceaccount/ca.crt
+     insecure_skip_verify: true  # Required with Minikube.
```
Comment (bobdonat) [inline pkg/resources/helper.go:238]: Do we care about Minkube?

File: THIRD_PARTY_LICENSES.txt [hunk 3]
```diff
@@ -5371,434 +5371,6 @@ LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING
...
-
```
Comment (bobdonat) [inline THIRD_PARTY_LICENSES.txt:5375]: @pfmackin should this be removed from  THIRD_PARTY_LICENSES.txt

File: pkg/resources/helper.go (hunk 2 above)
Comment (shihchang) [reply pkg/resources/helper.go]: no, removed the comment. tested with oke.

File: THIRD_PARTY_LICENSES.txt (hunk 3 above)
Comment (shihchang) [reply THIRD_PARTY_LICENSES.txt]: reverted, and build passed "Third Party License Check" this time.

File: THIRD_PARTY_LICENSES.txt (hunk 1 above)
Comment (shihchang) [reply THIRD_PARTY_LICENSES.txt]: reverted, and build passed "Third Party License Check" this time.

**Review by bobdonat — APPROVED** [review]

**Pull request merged**